lisa/env.py

```diff
@@ -748,6 +744,23 @@ def disable_idle_states(self):
             for domain in self.target.cpufreq.iter_domains():
                 self.target.cpuidle.enable_all(domain[0])
 
+    def get_emeter(self, res_dir=None):
```

Comment:
While I'm thinking about it, we need to go through the `EnergyMeter` class and remove the homemade classes that could be replaced by their devlib equivalents (e.g. ACME or hwmon).